### PR TITLE
first version of changes for icons in forum

### PR DESCRIPTION
**Pull request by weiji-li**
Just to test whether my small changes can work.

### CODE REVIEW

**Comment by VictorLi718** [comment]
I changed the icons, but I don't know which are the best so I just remove the circle around them and adjust the position. Also for the button for scrolling to top, I think change the opacity can help a lot(which is pretty easy to do). And when you hover on it, it will become 100% opacity and appear well.

**Comment by bmcutler** [comment]
Can we use the same version of the green check we use on the viewed grade column (from the instructor page click on grade or regrade for any released assignment. 

screen shot 2018-09-29 at 7 05 37 am

Let's try the icons without a black border?

**Comment by VictorLi718** [comment]
Ok, I will try. And what about the exclamation mark?

**Comment by bmcutler** [comment]
We're trying to use font awesome for all of our icons.  

Maybe the check and x for viewed grade are not font awesome (that should be changed too), so maybe don't use specifically those shapes, but make them look similar (no black outline, and large enough to be clearly visible on the forum).

As I said in #2800, please switch the exclamation point to be a question mark (it is an unresolved question after all)

https://fontawesome.com/icons/question?style=solid
https://fontawesome.com/icons/check?style=solid

**Comment by VictorLi718** [comment]
image
I make a change again, making it a bit larger and without black border. But maybe for question icon, yellow is not the best even if I use the same color as the border of selected answer in discussion forum

**Comment by bmcutler** [comment]
Try an orange color for the question mark.  Maybe ff9933 or ff6600?

**Comment by VictorLi718** [comment]
image
See this. I think it's better?

**Comment by bmcutler** [comment]
Yes, I think that looks good.  But this is not what the code does yet...

**Comment by VictorLi718** [comment]
Sorry, I just make a new push, with orange color. And I just see that maybe the "gold" color, which is used for the starred topic is also good.

File: site/app/views/forum/ForumThreadView.php [hunk 1]
```diff
@@ -624,14 +624,19 @@ public function displayThreadList($threads, $filtering, &$activeThreadAnnounceme
...
+								$fa_icon = "fa-check";
+								$fa_color = "5cb85c";
+								$fa_margin_right = "0px";
+								$fa_font_size = "1.5em";
+
 							} else {
-								$fa_icon = "fa-exclamation-circle";
-								$fa_color = "yellow";
+								$fa_icon = "fa-question";
+								$fa_color = "ff8000";
```
Comment (andrewaikens87) [inline site/app/views/forum/ForumThreadView.php]: Please include a # before the color hex.

File: site/app/views/forum/ForumThreadView.php [hunk 2]
```diff
@@ -624,14 +624,19 @@ public function displayThreadList($threads, $filtering, &$activeThreadAnnounceme
                         }
 						if($thread['status'] !=0) {
 							if($thread['status'] == 1) {
-								$fa_icon = "fa-check-circle";
-								$fa_color = "palegreen";
+								$fa_icon = "fa-check";
+								$fa_color = "5cb85c";
```
Comment (andrewaikens87) [inline site/app/views/forum/ForumThreadView.php]: Please include a # before the color hex.

Comment (VictorLi718) [reply site/app/views/forum/ForumThreadView.php]: Sorry, my bad. I have '#' before color hex value now.

File: site/app/views/forum/ForumThreadView.php [hunk 3]
```diff
@@ -624,14 +624,19 @@ public function displayThreadList($threads, $filtering, &$activeThreadAnnounceme
...
-								$fa_color = "yellow";
+								$fa_icon = "fa-question";
+								$fa_color = "#ff8000";
+								$fa_margin_right = "5px";
+								$fa_font_size = "1.8em";
 							}
 							$return .= <<<HTML
-							<i class="fa ${fa_icon}" style="padding-left:3px;position:relative; float:right; display:inline-block; color:${fa_color}; -webkit-text-stroke-width: 1px;
+							<i class="fa ${fa_icon}" style="margin-right:${fa_margin_right}; padding-left:3px; position:relative; float:right; display:inline-block; color:${fa_color}; font-size:${fa_font_size}; -webkit-text-stroke-width: 0px;
     -webkit-text-stroke-color: black;" aria-hidden="true"></i>
```
Comment (andrewaikens87) [inline site/app/views/forum/ForumThreadView.php]: Please remove the stroke css instead of setting it to zero.

**Pull request merged**